dandi/consts.py

```diff
@@ -141,7 +141,17 @@ def urls(self) -> Iterator[str]:
         "linc-staging",
         "https://staging.lincbrain.org",
         "https://staging-api.lincbrain.org/api",
-    )
+    ),
+    "ember": DandiInstance(
+        "ember",
+        "https://ember-archive.org",
+        "https://api.ember-archive.org/api",
+    ),
+    "ember-staging": DandiInstance(
```

Comment:
note to keep in mind: within DANDI we are talking about renaming to `testing` or alike (attn @satra), so we better do it "right" to start with in EMBER

Reply:
PR in progress here: https://github.com/dandi/dandi-archive/pull/2139